src/qos_host_primitives/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "qos_host_primitives"
```

Comment:
I feel strongly we do not want this to be a qos lib. This is very opinionated to the way turnkey writes enclave hosts (and we can publish it) - but I don't think qos should be opinionated about that